### DISCUSSION TITLE
Some user friendly tips for Incremental Clustering code

### DISCUSSION THREAD

**Discussion by Anurag19101996** [uw-ssec/MAWpy#40] [General]
a. If incremental clustering is going as the first step of the workflow, do we have any warnings or messages prompted to notify that they must set the time duration to 0? 
b. Could additional comments be added, particularly for the lines and loops, to provide more detail? While the current comments are sufficient for me to understand the code, more detailed comments would be beneficial for those who may not be as proficient in reading code.
c. Additionally, could a flow process chart (similar to pseudocode for the algorithm) be added at the beginning of the code? We need to confirm whether THINK Lab or SSEC will add it.

@qzchen-uw @gracejia513 @anujsinha3 @carlosgjs

**Comment by anujsinha3** [comment]
Adding pictorial markdowns/flow process chart for 'readthedocs' under the docs folder might be a better way to increase user clarity of provided functionalities.
